Resources/Prototypes/_Harmony/Entities/Mobs/Player/animals.yml

```
@@ -0,0 +1,44 @@
+﻿- type: entity
+  name: Rampaging Gorilla
```

Names should not be capitalized
```suggestion
  name: rampaging gorilla
```